package.json

```diff
@@ -1,27 +1,35 @@
 {
-  "author": "Roman Shtylman <shtylman@gmail.com>",
   "name": "localtunnel",
-  "description": "expose localhost to the world",
-  "version": "1.9.2",
+  "description": "Expose localhost to the world",
+  "version": "2.0.0",
```

Comment:
The version bump should happen at the same time as the changelog goes from UNRELEASED - so here the version should remain as before. I wish package.json supported UNRELEASED but it does not - so it is hard to indicate that this is neither version 1.9.2 nor version 2.0.0

Reply:
@ghengeveld This is the last outstanding item - please make it `2.0.0-pre` and I will bump this to 2.0.0 when I release.